packages/kbn-optimizer/limits.yml

```diff
@@ -205,6 +205,6 @@ pageLoadAssetSize:
   visTypeXy: 32342
   visualizations: 38375
   watcher: 10485
-  workflowsExtensions: 2554
+  workflowsExtensions: 61264
```

Comment:
Were we able to look into bringing this back down?

Reply:
We will optimize it later. We know that it's not ideal thing.
cc: @semd